api/src/main/java/org/openmrs/api/db/hibernate/HibernateDiagnosisDAO.java

```diff
@@ -14,22 +14,21 @@
 
 import org.hibernate.Query;
 import org.hibernate.SessionFactory;
-import org.openmrs.Diagnosis;
-import org.openmrs.Encounter;
-import org.openmrs.Patient;
+import org.openmrs.*;
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports

Reply:
Thanks - I think my IDE was not configured properly. Apologies.